js/utils/trumbowyg.js

```diff
@@ -0,0 +1,29 @@
+import $ from 'jquery';
+import 'trumbowyg';
+
+$.trumbowyg.svgPath = '../node_modules/trumbowyg/dist/ui/icons.svg';
+
+export const editorOptions = {
+  btns: [
+    ['formatting'],
+    ['bold', 'italic'],
+    ['link'],
+    ['insertImage'],
+    'btnGrp-justify',
+    'btnGrp-lists',
+  ],
+};
+
+export function installRichEditor(attachPoint) {
```

Comment:
I'm not sure if this needs to necessarily be changed, but flagging it as a concern. When Backbone View's call this method, they should really only pass in the attachPoint as an element (jQuery or otherwise) that was scoped to their view.

The risk with passing it in as a selector is a.) The whole DOM is needlessly being searched rather than a subset of it (i.e. the el of your view) b.) related to A, there is a risk of finding an element a different view, for example, if you passed in a somewhat simple selector of `.description`, there may very well be other elements in the dom matching that.

So... we can either leave it as it is and leave it up to the dev using it (and the reviewers reviewing it) to be disciplined or we could also integrate this specific function into the baseVw and have the selector be scoped to the view `const attach = this.$(attachPoint);`. I tend to lean as the latter option being safer.